scripts/build/index.ts

```diff
@@ -307,6 +307,10 @@ export const createManifest = (): any => {
         types: './import.d.mts',
         default: './legacynode.mjs',
       },
+      './types': {
+        types: './types.d.ts',
+        default: './build/types.d.ts',
```

Comment:
This is generating the file that is placed within the `build` folder already, so there isn't a `build` subfolder here:

```suggestion
        default: './types.d.ts',
```